com.adsbynimbus.nimbus/Runtime/Scripts/Nimbus.Internal/Interceptor/ThirdPartyDemand/Meta/MetaAndroid.cs

```diff
@@ -25,10 +25,6 @@ public BidRequest ModifyRequest(BidRequest bidRequest, string data) {
 			bidRequest.User.Ext.FacebookBuyerId = data;
 			if (bidRequest.Imp.Length > 0) {
 				bidRequest.Imp[0].Ext.FacebookAppId = _appID;
-				if (_testMode)
-				{
-					bidRequest.Imp[0].Ext.MetaTestAdType = "IMG_16_9_LINK";
```

Comment:
This code should stay in, if the Nimbus SDK test mode is enabled we want to send along the test ad type.